server/src/main/java/org/opensearch/index/merge/UnreferencedFileCleanUpStats.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.index.merge;
+
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+import org.opensearch.core.common.io.stream.Writeable;
+import org.opensearch.core.xcontent.ToXContentFragment;
+import org.opensearch.core.xcontent.XContentBuilder;
+
+import java.io.IOException;
+
+/**
+ * Stores stats about unreferenced file cleanup due to segment merge failure.
+ *
+ * @opensearch.internal
+ */
+public class UnreferencedFileCleanUpStats implements Writeable, ToXContentFragment {
```

Comment:
Do we plan to add more fields under `UnreferencedFileCleanUpStats`? Or this is just to make it future-proof?

Reply:
On second thought it seems we may not need to add any new field here. Added this metric can as a separate field to merge stats and removing the class